src/test/java/levy/dictionary/EnglishDictionaryTest.java

```diff
@@ -0,0 +1,41 @@
+package levy.dictionary;
+import com.opencsv.exceptions.CsvValidationException;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class EnglishDictionaryTest {
+
+    @Test
+    public void testGetDefinition() {
```

Comment:
Do not add the word "test" to your test methods.
This is a very old way of writing tests that isn't necessary anymore.